Core/src/main/java/com/paypal/android/core/PayPalSDKError.kt

```diff
@@ -3,7 +3,7 @@ package com.paypal.android.core
 /**
  * Class used to describe PayPal errors when they occur.
  */
-data class PayPalSDKError(
+open class PayPalSDKError(
```

Comment:
Does `open data class` still work here?

Reply:
`data` classes are not inheritable. I think this does promt a discussion on how we are going to model our errors, but its out of the scope of this PR